### ISSUE TITLE
Display ad markers when using the Leanback UI via ExoPlayer's Leanback extension

### ISSUE THREAD

**Issue by SakthivelMuruganS** [google/ExoPlayer#6695]
I want to display markers for ads in Exoplayer using the Leanback Adapter and Glue. 
I've used Player Adapter & PlaybackTransportControlGlue (androidx.leanback.media) to stream the content. I'm able to load ads using Ima ads Loader in the video but I want to show the ad marker in lean-back seek bar like Exo player default time bar for different types of ad roll (pre-roll, mid-roll & post-roll).

**Comment by ojw28** [comment]
I don't think the leanback library supports ad markers (else there would be a way to expose them [here](https://developer.android.com/reference/android/support/v17/leanback/media/PlayerAdapter)). So I think your options are to file a feature request [here](http://issuetracker.google.com/issues?q=componentid:460758%20status:open), or build your own player controls UI instead.

If you do file a feature request, please let us know (by replying here). I'd be interested to follow/nudge it along, since this seems like a fairly basic shortcoming of the leanback UI.

**Comment by ojw28** [comment]
Marking this as "needs for information", because you'll need to file a feature request on the leanback library asking them to support ad markers. Without that, this issue is blocked on a feature request in another library that no one has requested.

Please respond here with a link to the leanback feature request if you make one. Else this issue will be automatically closed.

**Comment by SakthivelMuruganS** [comment]
I've filed a feature request and please find the feature request link in below
https://github.com/google/ExoPlayer/issues/6789

Kindly let us know the update on this.
Thank you.

**Comment by ojw28** [comment]
You need to file a feature request on the Leanback Library, not on ExoPlayer. As in my initial response, you should do this [here](http://issuetracker.google.com/issues?q=componentid:460758%20status:open).

**Comment by SakthivelMuruganS** [comment]
Can you tell me what is the component in feature request option??

**Comment by ojw28** [comment]
The link should already take you to the right component (componentid:460758). You can click the drop arrow on the "create issue" button and click "in same component".

**Comment by SakthivelMuruganS** [comment]
Yes I've filed a request in this and Thanks for your kind support

**Comment by ojw28** [comment]
Leanback feature request: http://issuetracker.google.com/issues/146537025
